jobs/migrateboard.go

```diff
@@ -12,16 +12,19 @@ import (
 	"github.com/meifamily/ptt-alertor/models/pushsum"
 )
 
-const preBoard = "movies"
-const postBoard = "movie"
-
 type migrateBoard struct{}
 
 func NewMigrateBoard() *migrateBoard {
 	return &migrateBoard{}
 }
 
-func (migrateBoard) Run() {
+func (m migrateBoard) Run(boardMap map[string]string) {
```

Comment:
We use this scheduled job library, to reuse the code for cronjob, need make `Run()` as non parameter method
> https://github.com/robfig/cron